pkg/command/map_string.go

```diff
@@ -36,6 +36,18 @@ func GetStringMapStringE(vp *viper.Viper, key string) (map[string]string, error)
 	if data == nil {
 		return map[string]string{}, nil
 	}
+
+	return ToStringMapStringE(vp.Get(key))
+}
+
+// ToStringMapStringE casts an interface to a map[string]string type. The underlying
```

Comment:
would be great to have this change reflected in the cilium docs -> https://docs.cilium.io/en/latest/contributing/development/hive/#config

especially because of the special encoding (json & kv fallback)

Reply:
I've added an [additional commit](https://github.com/cilium/cilium/pull/25643/commits/372d3f295c2480819dad2b8d6fc3274e43c2abc6) which extends the example and comments on how parsing is performed. PTAL